docs/architektura.md

```diff
@@ -22,9 +22,19 @@
 - Mikrokontroler w doniczce komunikuje się bezprzewodowo przez Wi-Fi z hostem na Raspberry Pi.
 - Dane z sensorów są przesyłane do Raspberry Pi, które przetwarza te informacje i podejmuje decyzje na podstawie zaprogramowanych algorytmów.
 
-#### Schemat łączności i wybudzania serwer <-> klient
+## UML systemu
```

Comment:
Może 'Schemat UML systemu', nie każdy wie co to jesy

Reply:
🫡